aws/PAS-AIO-dr-Deployment.json

```diff
@@ -1195,6 +1200,11 @@
                         "waitAfterCompletion":"0",
                         "ignoreErrors":"false"
                      },
+                     "03-CheckVaultServiceStatus": {
```

Comment:
In DR PrivateArk Server should be stopped

Reply:
Done